streams/src/main/java/org/apache/kafka/streams/kstream/internals/MaterializedInternal.java

```diff
@@ -56,6 +69,10 @@ public String storeName() {
         return storeName;
     }
 
+    public StoreType storeType() {
+        return storeType == null ? StoreType.ROCKS_DB : storeType;
```

Comment:
The `storeType` should never be null since it's always initialized at construction time right?

Reply:
You're right! We can set default to rocksDB if `null`, and we don't need this null check here. Updated.